_data/events/Bellevue.json

```diff
@@ -1,11 +1,11 @@
 {
   "title": "Global Day of Code Retreat 2017 - Eastside",
-  "url": "http://bit.ly/gdcr17Eastside",
+  "url": "http://bit.ly/gdcr17eastside",
```

Comment:
Please add the resolved link here. We only use it in links, so it'd be nice for people to know where the link will lead them to.